hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestStoreScanner.java

```diff
@@ -97,7 +120,20 @@
   private static final int CELL_GRID_BLOCK3_BOUNDARY = 11;
   private static final int CELL_GRID_BLOCK4_BOUNDARY = 15;
   private static final int CELL_GRID_BLOCK5_BOUNDARY = 19;
-
+  private final static byte[] fam = Bytes.toBytes("cf_1");
+
+  @BeforeClass
+  public static void setUp() throws Exception {
```

Comment:
Can we use a new test for this? The TestStoreScanner is already complicated enough...

Reply:
Sure. will move the test to a new class